lms/djangoapps/verify_student/migrations/0009_auto__change_softwaresecurephotoverification_window_id_default_none.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import connection
+
+class Migration(SchemaMigration):
+
+    def forwards(self):
+
+        # Changing field 'SoftwareSecurePhotoVerification.window'. Setting its default value to None
+        cursor = connection.cursor()
+        cursor.execute('ALTER TABLE verify_student_softwaresecurephotoverification CHANGE `window_id` `window_id` int(11) DEFAULT NULL;')
```

Comment:
This is breaking on the acceptance tests that use SQLite.  Ideally migrations should be independent of the database backend.  Looking at the South APIs, though, I'm not seeing a way to do this.

It's suboptimal, but I think our best option is to run this migration only on MySQL, [like this](https://github.com/edx/edx-ora2/blob/c56853ae0a0846d70ac98bf2688ce5490bd46977/openassessment/assessment/migrations/0018_auto__add_field_assessmentpart_criterion.py#L12-L17).

@feanil what do you think?  Is there a better way to do this that I'm not seeing?